dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -285,9 +284,7 @@ private String getOrgUnitModeViolation(EventOperationParams params, User user, P
           : "Current user is not authorized to query across all organisation units";
       case ACCESSIBLE -> getAccessibleScopeValidation(user, program);
       case CAPTURE -> getCaptureScopeValidation(user);
-      case CHILDREN, SELECTED, DESCENDANTS -> params.getOrgUnitUid() == null
-          ? "Organisation unit is required for orgUnitMode: " + params.getOrgUnitMode()
-          : null;
+      default -> null;
```

Comment:
Shouldn't we have here now the validation for `ALL` mode?

userCanSearchOuModeALL(user)

Reply:
that's in the line 282, I can't put it here because CHILDREN, SELECTED and DESCENDANTS would be affected.